scripts/_helpers_dist.py

```diff
@@ -93,6 +103,89 @@ def configure_logging(snakemake, skip_handlers=False):
     logging.basicConfig(**kwargs)
 
 
+def create_country_list(input, iso_coding=True):
```

Comment:
This function could be imported as the previous ones:
create_country_list = pe_helpers.create_country_list